Session/Settings/SessionProSettings/SessionProSettingsViewModel+Database.swift

```diff
@@ -0,0 +1,13 @@
+// Copyright © 2025 Rangeproof Pty Ltd. All rights reserved.
+
+import Foundation
+import GRDB
+import DifferenceKit
+import SessionUtilitiesKit
+
+public extension KeyValueStore.IntKey {
+    static let groupsUpgradedCounter: KeyValueStore.IntKey = "groupsUpgradedCounter"
+    static let pinnedConversationsCounter: KeyValueStore.IntKey = "pinnedConversationsCounter"
+    static let proBadgesSentCounter: KeyValueStore.IntKey = "proBadgesSentCounter"
+    static let longerMessagesSentCounter: KeyValueStore.IntKey = "longerMessagesSentCounter"
+}
```

Comment:
Are we meant to be running a query to get a "live" state for these or incrementing a counter? (both behaviours have edge-cases where the values are in-accurate so I just wanted to double check what the expected "bugs" are 😅)

Reply:
I think we want the historic data for upgraded groups (not implemented in this stage), pro badges sent, and longer messages sent according to the PRD. So those 3 counters are necessary because a live state may not be accurate due to message deletion. I'll remove the pinned convo counter though, because we want a live state of that one from LibSession.